utils/recursiveDependencyFetchWorker.js

```diff
@@ -0,0 +1,147 @@
+/* eslint-disable no-eval */
+const UNPKG = 'https://unpkg.com/';
+
+const fileNameRegEx = /\/[^\/@]+[\.][^\/]+$/;
+
+// Hack to get ESM into webworker as 21/Nov/2019 ESM is not supported by webworkers
```

Comment:
can we use [importScripts](https://developer.mozilla.org/en-US/docs/Web/API/WorkerGlobalScope/importScripts) for this?

Reply:
importScripts won't work with the es module?